server/prerender.js

```diff
@@ -1,18 +1,24 @@
 var prerenderio = Npm.require('prerender-node');
 var token;
 var serviceUrl;
+var protocol;
 var settings = Meteor.settings.PrerenderIO;
 
 // token
 token = process.env.PRERENDERIO_TOKEN || (settings && settings.token);
 
+
 // service url (support `prerenderServiceUrl` (for historical reasons) and `serviceUrl`)
 serviceUrl = settings && (settings.prerenderServiceUrl || settings.serviceUrl);
 serviceUrl = process.env.PRERENDERIO_SERVICE_URL || serviceUrl;
 
+// protocol
+protocol = process.env.PRERENDERIO_PROTOCOL || (settings && settings.protocol);
```

Comment:
This should default to `http` to not break current behavior. Otherwise everyone using the plugin has to set this. 

Or you could use `if(protocol) prerenderio.set('prerenderProtocol', protocol);` below. Actually, I'd like that even more :)